utils/di.go

```diff
@@ -30,10 +31,21 @@ var (
 
 // init loads default implementations of common components for unit tests' purpose.
 func init() {
+	ResetDefaults()
+}
+
+// ResetDefaults reset default config, logger and metrics settings
+func ResetDefaults() {
 	logger = common.NewLoggerFactory().GetDefaultLogger()
 	queryLogger = common.NewLoggerFactory().GetDefaultLogger()
 	scope := tally.NewTestScope("test", nil)
```

Comment:
why test scope?

Reply:
these are default settings used in tests mostly

in production we set using utils.Init()